.travis.yml

```diff
@@ -8,11 +8,11 @@ python:
   - 3.7
   - pypy3
 install:
-  - pip install pytest pytest-cov flake8
+  - pip install "pytest>=4.6,<5" "pytest-cov>=2.7,<3" "flake8>=3.7,<4"
```

Comment:
Why not use test deps here (from the setup.py)?

Reply:
Changed, see above.